src/main/java/wolox/training/integration/dto/Author.java

```diff
@@ -0,0 +1,57 @@
+
+package wolox.training.integration.dto;
+
+import java.util.HashMap;
+import java.util.Map;
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+    "url",
+    "name"
+})
```

Comment:
`@JsonPropertyOrder` is not necessary but if you want to keep it use constants in the values